fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -306,7 +306,7 @@ public RdfStream apply(final FedoraResource child) {
             httpTripleUtil.addHttpComponentModelsForResourceToStream(rdfStream, resource(), uriInfo, translator());
         }
 
-        return rdfStream.map(deskolemizer());
+        return rdfStream;
```

Comment:
If you make this change, then at what point is the necessary deskolemization actually going to happen?